### PR TITLE
Update project list routes

### PR DESCRIPTION
**Pull request by sirmspencer**

### CODE REVIEW

File: src/sql/functions/project.sql [hunk 1]
```diff
@@ -777,13 +777,10 @@ CREATE OR REPLACE FUNCTION valid_boundary(_boundary geometry)
...
-        WHERE ST_IsValid(_boundary)
-            AND NOT (ST_XMax(_boundary) > 180
-                OR ST_XMin(_boundary) < -180
-                OR ST_YMax(_boundary) > 90
-                OR ST_YMin(_boundary) < -90
-                OR ST_XMax(_boundary) <= ST_XMin(_boundary)
-                OR ST_YMax(_boundary) <= ST_YMin(_boundary))
+        WHERE _boundary IS NOT NULL
+            AND ST_Contains(ST_MakeEnvelope(-180, -90, 180, 90, 4326), _boundary)
+            AND ST_XMax(_boundary) > ST_XMin(_boundary)
```
Comment (rfontanarosa) [inline src/sql/functions/project.sql:782]:
```suggestion
            AND ST_XMax(_boundary) > ST_XMin(_boundary)
```

File: src/sql/functions/project.sql [hunk 2]
```diff
@@ -777,13 +777,10 @@ CREATE OR REPLACE FUNCTION valid_boundary(_boundary geometry)
...
+            AND ST_YMax(_boundary) > ST_YMin(_boundary)
```
Comment (rfontanarosa) [inline src/sql/functions/project.sql:783]:
```suggestion
            AND ST_YMax(_boundary) > ST_YMin(_boundary)
```
why no more <=?

File: src/sql/changes/update_2021-01-15_sql_clean.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+DROP TYPE project_return;
```
Comment (rfontanarosa) [inline src/sql/changes/update_2021-01-15_sql_clean.sql:1]: where are all the other changes?

Comment (sirmspencer) [reply src/sql/changes/update_2021-01-15_sql_clean.sql:1]: Functions are automatically dropped and recreated now.  Did I miss any changes that are not just changes to functions?

File: src/sql/functions/project.sql (hunk 1 above)
Comment (sirmspencer) [reply src/sql/functions/project.sql:782]: Min and max cant be the same.  ST_IsValid was catching that before so the logic was fine before.

File: src/sql/functions/project.sql (hunk 2 above)
Comment (sirmspencer) [reply src/sql/functions/project.sql:783]: Min and max cant be the same.  ST_IsValid was catching that before so the logic was fine before.

Comment (sirmspencer) [reply src/sql/functions/project.sql:783]: Min and max cant be the same.  ST_IsValid was catching that before so the logic was fine before.

File: src/sql/changes/update_2021-01-15_sql_clean.sql (hunk 3 above)
Comment (sirmspencer) [reply src/sql/changes/update_2021-01-15_sql_clean.sql:155]: I added `ROP VIEW IF EXISTS project_boundary CASCADE;`

**Pull request merged**